packages/components/src/card/card/component.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import { css } from 'emotion';
+
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { contextConnect, ContextSystemProvider } from '../../ui/context';
+import { Elevation } from '../../elevation';
+import { View } from '../../view';
+import * as styles from '../styles';
+import { useCard } from './hook';
+import CONFIG from '../../utils/config-values';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').Props, 'div'>} props
+ * @param {import('react').Ref<any>} forwardedRef
+ */
+function Card( props, forwardedRef ) {
+	const {
+		children,
+		elevation,
+		isBorderless,
+		isRounded,
+		size,
+		...otherProps
+	} = useCard( props );
+	const elevationBorderRadius = isRounded ? CONFIG.cardBorderRadius : 0;
+
+	const elevationClassName = useMemo(
+		() => css( { borderRadius: elevationBorderRadius } ),
+		[ elevationBorderRadius ]
+	);
+
+	const contextProviderValue = useMemo( () => {
+		const contexProps = {
+			size,
+			isBorderless,
+		};
+		return {
+			CardBody: contexProps,
+			CardHeader: contexProps,
+			CardFooter: contexProps,
+		};
```

Comment:
A small typo 🙂 

```suggestion
		const contextProps = {
			size,
			isBorderless,
		};
		return {
			CardBody: contextProps,
			CardHeader: contextProps,
			CardFooter: contextProps,
		};
```

Reply:
Nice catch!  Fixed in https://github.com/WordPress/gutenberg/pull/32566/commits/c001f9eeeafce3eb280744a5ca0cd08ab0105532